src/components/nav.css

```diff
@@ -0,0 +1,16 @@
+nav {
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  text-align: center;
+}
+
+ul {
+  list-style: none;
+}
+
+li {
+  display: inline;
+  padding: 10px;
+}
```

Comment:
I love that you’ve kept the styling super minimal at this stage of the game! 👏👏👏